packages/react-utilities/src/compose/getSlots.ts

```diff
@@ -5,27 +5,27 @@ import { omit } from '../utils/omit';
 import type {
   AsIntrinsicElement,
   ComponentState,
-  ObjectShorthandProps,
-  ObjectShorthandPropsRecord,
-  ShorthandRenderFunction,
+  SlotProps as ObjectSlotProps,
```

Comment:
It would probably be better to rename the local SlotProps instead of renaming the import.

Reply:
Done!